React/CoreModules/RCTAlertController.m

```diff
@@ -35,6 +35,9 @@ - (void)show:(BOOL)animated completion:(void (^)(void))completion
 
 - (void)hide
 {
+  [_alertWindow resignKeyWindow];
+  [_alertWindow setHidden: YES];
+  _alertWindow.rootViewController = nil;
```

Comment:
Wondering if `_alertWindow.hidden = YES` is enough to get the expected behavior

Reply:
Good catch, let me experiment

Reply:
Maybe this will work
```
  if (@available(iOS 13.0, *)) {
	  _alertWindow.windowScene = nil;
  }
```
https://developer.apple.com/documentation/uikit/uiwindow/3197962-windowscene?language=objc

Reply:
@danibonilha thanks for your help.  setting hidden was enough.  I confirmed using Xcode debug view hierarchy

![Screen Shot 2021-10-01 at 3 26 09 PM](https://user-images.githubusercontent.com/535096/135682313-4c3e63a2-7c2f-41a6-b65e-4bf71fc4eb72.png)

Reply:
(If it hadn't worked you'd see another UIWindow hanging out in the scene)

Reply:
For reference, here you can see the orphaned window (prior to applying this fix)
![Screen Shot 2021-10-01 at 3 29 10 PM](https://user-images.githubusercontent.com/535096/135682619-b7e6ce57-0766-4e6a-a17d-b9de5e46a717.png)